cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -41,6 +41,12 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
+    app.kubernetes.io/name: rook-ceph
+    app.kubernetes.io/instance: rook-ceph
+    app.kubernetes.io/component: csi
+    app.kubernetes.io/part-of: rook-ceph
+    app.kubernetes.io/managed-by: rook-ceph-operator
```

Comment:
The resources in common.yaml are really created and managed by the admin, rather than Rook. How about we leave off these two labels for the resources in common.yaml? Or perhaps we just leave them blank.